README.md

```diff
@@ -256,6 +256,7 @@ the table below for reference.
 
 | Variable | Description | Default | Required |
 | --- | --- | --- | --- |
+| AV_SCAN_PATH | Path where the file to be scanned will be temporarily stored | /tmp | No |
```

Comment:
I guess you're passing in the EFS location as the `AV_SCAN_PATH`?  Is that configuration configured manually, or should there be a matching Cloud Formation PR to go along with these changes?

Reply:
We are going to manage that manually for now. Given this isn't tied to a CF stack in the same way as other our AWS Resources, and that the large Lambda archive files are not a great fit for CloudFormation.